### ISSUE TITLE
Api state

### ISSUE THREAD

**Issue by DonsWayo** [brawlstarsAPI/api#7]
Hi, i see the api not working, have any plans to work again?

**Comment by raphaelkieling** [comment]
I did want work with this api, but what's happen? Any update?

**Comment by MariusMeiners** [comment]
i dont think it is being actively developed ? An update by the repo holder would be great

**Comment by MariusMeiners** [comment]
i dont think it is being actively developed ? An update by the repo holder would be great...

**Comment by raphaelkieling** [comment]
That would be great, but he is gone

**Comment by raphaelkieling** [comment]
Did someone  use this [brawstar api in python](https://github.com/umbresp/brawlstars)?

**Comment by zihadmahiuddin** [comment]
Hello guys,
You guys can use my API if you want.
Thanks.
